doc/usermanuals/DDG4/sections/Setup.tex

```diff
@@ -597,7 +597,7 @@ \subsection{Setting up DDG4 using Python}
 
   # Setup particle gun
   gun = DDG4.GeneratorAction(kernel,"Geant4ParticleGun/Gun")
-  gun.energy   = 0.5*GeV
+  gun.Energy   = 0.5*GeV
```

Comment:
A bit unusual to change Case only for the energy.

Reply:
Also how about
```suggestion
  gun.momentumMin = 0.5*GeV
  gun.momentumMax = 0.5*GeV
```
if energy is to be deprecated?

Reply:
No, I no longer want to deprecate energy. That will then still allow a simple way to set a fixed energy, and it will always be fixed (again).

Reply:
The uppercase is because the `Energy` property was also set in one of the `Geant4...cpp` files, but actually simpler to allow both upper and lowercase. Because both are used in various examples, and maybe in the wild